tests/test_implicits.py

```diff
@@ -0,0 +1,90 @@
+from fast_depends import Depends
+
+from emp_agents.implicits import IgnoreDepends, inject, lazy_implicit  # , set_implicit
+from emp_agents.utils import get_function_schema
+
+
+class MyObject:
+    def __init__(self, value: int):
+        self.value = value
+
+
+def old_load_object() -> MyObject:
+    return MyObject(value=1)
+
+
+def new_load_object(a: int) -> MyObject:
+    print("A", a)
+    return MyObject(value=100)
+
+
+@inject
+def do_thing_with_object(
+    a: int,
+    b: int = 100,
+    obj: MyObject = IgnoreDepends(lazy_implicit("load_object")),
+):
+    print("AB", a, b)
+    assert isinstance(obj, MyObject)
+    return obj
+
+
+# def test_implicit_manager():
```

Comment:
I think we can do away with the lazy implicits and just use Providers and Scope in the fast_depends library